packages/plugin-expo-device/device.js

```diff
@@ -0,0 +1,52 @@
+const { Constants } = require('expo')
+const { Dimensions, Platform } = require('react-native')
+const { isoDate } = require('@bugsnag/core/lib/es-utils')
+const rnVersion = require('react-native/package.json').version
+
+module.exports = {
+  init: client => {
+    let orientation
+    const updateOrientation = () => {
+      const { height, width } = Dimensions.get('screen')
+      if (height > width) {
+        orientation = 'portrait'
+      } else if (height < width) {
+        orientation = 'landscape'
+      } else {
+        orientation = undefined
+      }
+    }
+    Dimensions.addEventListener('change', updateOrientation)
+
+    // get the initial orientation
+    updateOrientation()
+
+    client.device = {
+      ...client.device,
+      id: Constants.installationId,
+      manufacturer: Constants.platform.ios ? 'Apple' : undefined,
+      modelName: Constants.platform.ios ? Constants.platform.ios.model : undefined,
+      modelNumber: Constants.platform.ios ? Constants.platform.ios.platform : undefined,
+      osName: Platform.OS,
+      osVersion: Constants.platform.ios ? Constants.platform.ios.system : Constants.systemVersion,
```

Comment:
Is this correct?  Docs wise it seems the calls should be `Constants.platform.ios.systemVersion` and `Platform.Version`

Reply:
Ah, yeah you're right. I'll update this.

Did you also run it and log out payloads? This is unit tested, but all the external stuff is mocked, so like with this example if both the tests and impl are wrong in the same way we get a false positive. It would be good to have some payloads from different devices (even if they are simulators) eyeballed until we have proper e2e tests.